gapic-generator-java/src/test/java/com/google/api/generator/gapic/composer/ComposerTest.java

```diff
@@ -61,6 +62,7 @@ void initialSanityCheck() {
     assertTrue(context.containsServices());
   }
 
+  @Disabled
```

Comment:
IIUC, this is a manual fix and this `@Disabled` is added temporarily and will be removed after this update is merged?

Reply:
No, I was testing if adding `@Disabled` to this test is going to fix most of the issues or not. It proves not, we need to update integration tests, showcase tests, hermetic build tests and spring-cloud-gcp tests.  I'm going to update all the golden files in this repo for now to unblock the release, and create issues for improvements.